manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FujitsuClientComputingLimited.DeskUpdate
+PackageVersion: 5.2

```

```suggestion
PackageVersion: 5.2.40.0
```
This must match Control Panel.